bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/TemplateUtils.java

```diff
@@ -99,8 +106,9 @@ public ReadRowsRequest get() {
         }
 
         ReadHooks readHooks = new DefaultReadHooks();
-        ReadRowsRequest.Builder builder = Adapters.SCAN_ADAPTER.adapt(scan, readHooks);
-        cachedRequest = readHooks.applyPreSendHook(builder.build());
+        Query query = Query.create(options.getBigtableTableId().get());
+        Adapters.SCAN_ADAPTER.adapt(scan, readHooks, query);
+        cachedRequest = ((Query)readHooks.applyPreSendHook(query)).toProto(requestContext);
```

Comment:
Why do you need to cast `readHooks .applyPreSendHook` to a `Query`?

Reply:
It was showing up as an error in IDE, so added, must be because I hadn't run maven while making reference change as it is dependent on shaded jars. Would remove the unnecessary Cast.